recipes/resiprocate/all/conanfile.py

```diff
@@ -0,0 +1,78 @@
+import os
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+required_conan_version = ">=1.29.1"
+
+class ResiprocateConan(ConanFile):
+    name = "resiprocate"
+    description = "The project is dedicated to maintaining a complete, correct, and commercially usable implementation of SIP and a few related protocols. "
+    topics = ("sip", "voip", "communication", "signaling")
+    url = "https://github.com/conan-io/conan-center-index"
```

Comment:
```suggestion
    url = "https://github.com/resiprocate/resiprocate"
```

Reply:
The url is correct.
Homepage is used for the package.

Reply:
This field is managed by [this ](https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H027)rule, and is strict by content, we can not change it